app/views/movie/search.html.erb

```diff
@@ -1,11 +1,11 @@
 <h1>Search Results</h1>
```

Comment:
Curious what our final index page is going to look like since right now they are 2 separate html files

OH WAIT - I see that these files are super similar! Perhaps even identical. This is great, makes the update to index so much easier :)